projects/github-actions/repo-gardening/src/utils/openai/send-request.js

```diff
@@ -1,19 +1,19 @@
-const { getInput } = require( '@actions/core' );
+const { getInput, setFailed } = require( '@actions/core' );
 const OpenAI = require( 'openai' );
 const debug = require( '../debug' );
 
 /**
  * Send a message to OpenAI.
  *
- * @param {string} message - Message to send to OpenAI.
+ * @param {string} message        - Message to send to OpenAI.
+ * @param {string} responseFormat - Response format to use (plain by default, can be 'json_object').
  *
- * @return {Promise<string>} Promise resolving to the response from OpenAI.
+ * @return {Promise<string|void>} Promise resolving to the response from OpenAI, or void if an error occurred.
  */
-async function sendOpenAiRequest( message ) {
+async function sendOpenAiRequest( message, responseFormat = 'plain' ) {
 	const apiKey = getInput( 'openai_api_key' );
 	if ( ! apiKey ) {
-		debug( 'openai: Input openai_api_key is required but missing. Aborting.' );
-		return '';
+		setFailed( 'openai: Input openai_api_key is required but missing.' );
```

Comment:
Should there be a return here so it doesn't try to call OpenAI anyway?

Reply:
Done in 46ff82fbabd0efa071199c60e4fdcec8f5e86d8a